pkg/javascript/web/tests/memory_storage.rs

```diff
@@ -24,5 +24,4 @@ impl Tester<MemoryStorage> for MemoryTester {
 }
 
 generate_store_tests!(wasm_bindgen_test, MemoryTester);
-generate_dictionary_tests!(wasm_bindgen_test, MemoryTester);
```

Comment:
Is there any reason this is missing?

Reply:
We now removed `Metadata` trait so there does not exist `generate_dctionary_tests!` anymore, it is merged into `generate_store_tests!`